iot-hub/Samples/device/FileUploadSample/FileUploadSample.cs

```diff
@@ -48,8 +49,13 @@ public async Task RunSampleAsync()
 
                 // Note that other versions of the Azure Storage SDK can be used here. For the latest version, see
                 // https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage#azure-storage-libraries-for-net

```

Comment:
can use href here as well.

Reply:
this is a code comment, so href wouldn't work.